interpreter.go

```diff
@@ -373,7 +373,7 @@ func (i *interpreter) evaluate(a ast.Node, context *TraceContext) (value, error)
 	case *ast.ImportStr:
 		// TODO(sbarzowski) put this information in AST instead of getting it out of tracing data...
```

Comment:
It is in the AST isn't it?  In the location field

Reply:
Uh, of course.